### PR TITLE
feat: add rounded images snippet

### PR DESCRIPTION
**Pull request by Fadexz**
Added rounded images snippet + snippet image

### CODE REVIEW

**Comment by Fadexz** [comment]
Will I need lint to pass, my first commit so sorry if I messed it up

**Comment by theRealPadster** [comment]
> Will I need lint to pass? My first commit so sorry if I messed it up

Lint passed, you're good. We just might want to adjust how this is built since as it is right now, it will likely break when Spotify updates.

**Comment by rxri** [comment]
Never use past time in commit messages nor capitalize first word after scope. Thank you

File: src/resources/snippets.ts [hunk 1]
```diff
@@ -1,4 +1,10 @@
 export default [
+  {
+    "title": "Rounded Images",
+    "description": "Adds rounded corners to the cover art, playlist covers, and other images",
+    "code": "/* Cover Art Image (+ position fix) */\n .Foyk_HJx16yh22JYmQ56 {\n margin: 0 0 0 1px;\n border-radius: 6px;\n }\n \n /* Collapsed Cover Art Image */\n .cover-art-image {\n border-radius: 4px;\n }\n \n /* Playlist Header */\n /* Search Category Card + Image*/\n /* Local Files Card */\n .main-entityHeader-shadow,\n .tV9cjMpTPaykKsn2OVsw,\n .x-categoryCard-CategoryCard,\n .tlNAdRNP5lIeiD85nMcL {\n border-radius: 6px;\n }\n \n /* Track List Image */\n .main-trackList-rowImage {\n border-radius: 3px;\n }\n",
```
[the remark this answers is not in the document]
Comment (kyrie25) [reply src/resources/snippets.ts:5]: It is possible to use other selectors there in places of the hashes
But then again those specific class names hasn't changed in the past 30 versions

[the remark this answers is not in the document]
Comment (Fadexz) [reply src/resources/snippets.ts:5]: Yeah i've been running it for months but up to what you's think is best, apologies that i'm not the best with CSS as i'm learning

[the remark this answers is not in the document]
Comment (theRealPadster) [reply src/resources/snippets.ts:5]: I'd rather use other selectors just so there's less potential of breakage in the future. Unless the DOM structure is just as likely to change?

**Comment by Fadexz** [comment]
Let me know if you are happy with the changes I made
`/* Cover Art Image + Background */
.main-navBar-navBar > :nth-child(3) {
  margin: 0 0 0 1px;
  border-radius: 6px;
}

/* Collapsed Cover Art Image */
.cover-art-image {
  border-radius: 4px;
}

/* Playlist Header */
/* Search Category Card + Image*/
/* List Cards */
/* Local Files Card */
.main-entityHeader-shadow,
.x-categoryCard-image,
.main-cardImage-image,
.main-cardImage-imageWrapper,
.main-entityHeader-imagePlaceholder > div {
  border-radius: 6px;
}

/* Circled Artist Cards (force) */
.main-cardImage-circular {
    border-radius: 50% !important;
}

/* Track List Image */
.main-trackList-rowImage {
  border-radius: 3px;
}`
If something breaks I can fix it but should be pretty infrequent if it does happen

File: src/resources/snippets.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fadexz) [reply src/resources/snippets.ts:5]:
```
/* Expanded Cover Art Image (+ position fix) */
.main-navBar-navBar > :nth-child(3) {
  margin: 0 0 0 1px;
  border-radius: 6px;
}

/* Collapsed Cover Art Image */
.cover-art-image {
  border-radius: 4px;
}

/* Playlist Header */
/* Search Category Card Image */
/* List Cards */
/* Local Files Card */
.main-entityHeader-shadow,
.x-categoryCard-image,
.main-cardImage-image,
.main-cardImage-imageWrapper,
.main-entityHeader-imagePlaceholder > div {
  border-radius: 6px;
}

/* Circled Artist Cards (force) */
.main-cardImage-circular {
  border-radius: 50% !important;
}

/* Track List Image */
.main-trackList-rowImage {
  border-radius: 3px;
}
```

[the remark this answers is not in the document]
Comment (kyrie25) [reply src/resources/snippets.ts:5]: LGTM

[the remark this answers is not in the document]
Comment (Fadexz) [reply src/resources/snippets.ts:5]: Sweet, i'll try to do that

Quick question, what's the location for the snippet css?
Can't seem to find it anywhere, wanna know if I nuked all my snippets by accidentally installing the Microsoft Store version

[the remark this answers is not in the document]
Comment (theRealPadster) [reply src/resources/snippets.ts]: In the head or top or bottom of body I think? It's got a class on it that starts with "marketplace" I think, if you wanna ctrl+f for it

[the remark this answers is not in the document]
Comment (Fadexz) [reply src/resources/snippets.ts]: Unfortunately only the marketplace ones are in there, not the custom ones, have zero clue where else they would be stored

[the remark this answers is not in the document]
Comment (theRealPadster) [reply src/resources/snippets.ts]: You could just ctrl+f for a chunk of the snippet probably

[the remark this answers is not in the document]
Comment (Fadexz) [reply src/resources/snippets.ts]: After ages of looking but finding no file changes and looking through files anyway I decided to check the storage in DevTools and they were in there. So it's probably safe to say if I can't see them on the installed tab, they are likely lost

**Pull request merged**